qemu/tests/chardev_free_port.py

```diff
@@ -0,0 +1,46 @@
+from virttest import error_context, env_process
+from virttest import utils_misc
+
+from qemu.tests.virtio_console import add_chardev
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    qemu should try to find a free port value by to=<port> with unix socket and tcp options:
+    1) boot guest with socket with -qmp tcp:0:4444.
```

Comment:
1) boot guest with socket 'host=127.0.0.1,port=1234'
2) query chardev and check port number
3) boot another guest with socket 'host=127.0.0.1,port=1234,to=2234'
4) query chardev and check port number, should different from 2)